architecture/2-UserStories/2.2-ViewADS.yaml

```diff
@@ -0,0 +1,7 @@
+id: 2.2
+name: View ADS
+actor: Ground Station Operator 
+behavior: Determine location
```

Comment:
I'm thinking the behavior will need to be verbiage that describes a means to verify/validate that the OBC functions specific to ados (s.a., #138) are operating nominally.